package.json

```
@@ -457,6 +457,7 @@
     "@kbn/custom-branding-plugin": "link:x-pack/platform/plugins/private/custom_branding",
     "@kbn/custom-icons": "link:src/platform/packages/shared/kbn-custom-icons",
     "@kbn/custom-integrations-plugin": "link:src/platform/plugins/shared/custom_integrations",
+    "@kbn/dashboard_markdown_schemas": "link:src/platform/packages/private/kbn-dashboard_markdown_schemas",
```

this should be `dash-case`

```suggestion
    "@kbn/dashboard-markdown-schemas": "link:src/platform/packages/private/kbn-dashboard_markdown_schemas",
```